Library/Homebrew/brew.rb

```diff
@@ -123,7 +113,14 @@ def create_tap(cmd, tap_name)
 
     # Unset HOMEBREW_HELP to avoid confusing the tap
     ENV.delete("HOMEBREW_HELP") if help_flag
-    create_tap(cmd, possible_tap.name)
+    tap_commands = []
+    cgroup = Utils.popen_read("cat", "/proc/1/cgroup")
+    if !cgroup.include?("azpl_job") && !cgroup.include?("docker")
```

Comment:
Replaced the`unless .. ||` as `unless` with complex conditionals are hard to parse (for me at least).